experiments/conductor/cmd/runner/utilities.go

```diff
@@ -620,3 +635,24 @@ func processBranches(ctx context.Context, opts *RunnerOptions, branches []Branch
 		}
 	}
 }
+
+func runGoCmds(opts *RunnerOptions) error {
```

Comment:
nit: runGoModCmds.

Reply:
I was thinking we can add more here like goimports -w etc which changes imports or adds missing imports